src/ansys/fluent/core/solver/flobject.py

```diff
@@ -802,60 +801,55 @@ def get_cls(name, info, parent=None):
         else:
             cls = type(pname, (base,), dct)
 
-        children = info.get("children")
         taboo = set(dir(cls))
         taboo |= set(
             ["child_names", "command_names", "argument_names", "child_object_type"]
         )
-        if children:
-            taboo.add("child_names")
-            cls.child_names = []
-            for cname, cinfo in children.items():
+
+        doc = ""
+
+        def _process_cls_names(info_dict, names, write_doc=False):
+            nonlocal taboo
+            nonlocal cls
+
+            for cname, cinfo in info_dict.items():
                 ccls = get_cls(cname, cinfo, cls)
-                i = 0
                 ccls_name = ccls.__name__
+
+                i = 0
+                if write_doc:
+                    nonlocal doc
+                    th = ccls._state_type
+                    th = th.__name__ if hasattr(th, "__name__") else str(th)
+                    doc += f"    {ccls.__name__} : {th}\n"
+                    doc += f"        {ccls.__doc__}\n"
+
                 while ccls_name in taboo:
                     if i > 0:
                         ccls_name = ccls_name[: ccls_name.rfind("_")]
                     i += 1
                     ccls_name += f"_{str(i)}"
                 ccls.__name__ = ccls_name
                 # pylint: disable=no-member
```

Comment:
Is this line still needed?  You could check it with pylint...

Reply:
Hello @dnwillia-work,

Should I create new pyfluent and Fluent PR to implement following changes?  

1. removal of " # pylint: disable=no-member "
2. cls.command_names = list()
3. newkwds = dict()

Reply:
Yeah sure, when you have time, I did not realize you had already merged it.